src/components/pip-minimized/pip-minimized.tsx

```diff
@@ -24,18 +30,20 @@ export class PipMinimized extends Component<PIPMinimizedProps> {
     return (
       <Fragment>
         {!hideButtons && (
-          <div onMouseUp={show} role="button" className={styles.showContainer}>
-            <div className={styles.iconContainer}>
-              <Icon id="dualscreen-pip-minimized-show" height={16} width={16} path={icons.SHOW_ICON_PATH} />
-            </div>
-            Show
-          </div>
+          <Button onClick={show} className={styles.showContainer}>
+            <Fragment>
```

Comment:
Fragment is not needed if wrapped in the button

Reply:
Button takes only single child node, that's why Fragment needed here.